File: 100-Days-Series/Day 59/InvertedHalfPyramid.c
#include<stdio.h>
// Function to print Inverted Half Pyramid
void getPrintPyramid(int rows){
    for(int i=rows;i>=1;i--){
        for(int j=1;j<=i;j++){
            printf("%d ",j);
        }
        printf("\n");
    }
}
// Let's Start Here
void main(){
    int rows;
    printf("\nEnter the number of rows : ");
    scanf("%d",&rows);
    getPrintPyramid(rows);
}

File: 100-Days-Series/Day 59/InvertedHalfPyramid.js

// Arrow Function to print Inverted Half Pyramid
const getPrintPyramid=(rows)=>{
    for(let i=rows;i>=1;i--){
        line="";
        for(let j=1;j<=i;j++){
            line+=j+" ";
        }
        console.log(line);
    }
}

// Let's Start here
let rows=6;
getPrintPyramid(rows);